### ISSUE TITLE
Text definition of contributesTo is too narrow.

### ISSUE THREAD

**Issue by uscholdm** [semanticarts/gist#572]
In English, the meaning of 'contributes to' is very broad.  So it would be sensible to say that a degree in computer science contributes to a person developing programming skills.  The text definition of `contributesTo` is: 
- “The parts of a system contribute to the goal/ function of the whole system”

If you say a degree in computer science contributes to a person developing programming skills, there are no parts of any system per se, so it does not align with the text definition of contributes to. 

I’m saying that the definition of contributesTo is too narrowly focused on system-think, which is historical due to its origin in the network sub-gist.

PROPOSAL: 
- change the text definition to be more general 
- convert the current text definition into a `skos:example` 

ALTERNATIVE:
- instead of `contributesTo` use `facilitates`, which has similar meaning, but I think fits in more contexts.

**Comment by rjyounes** [comment]
I agree with broadening the definition of `contributesTo`. I think of examples as being more specific - individual cases - so I would just toss the current definition altogether.

**Comment by rjyounes** [comment]
This is subsumed by #534 but I'll leave it open so we make sure to take these points into consideration.

**Comment by jtdelany58** [comment]
@uscholdm <skos:definition rdf:datatype="&xsd;string">Something that has a share in bringing about a result.</skos:definition>

<skos:example rdf:datatype="&xsd;string">Example1: The parts of a system contribute to the goal/ function of the whole system.  Example 2: Fructose contributes to obesity in some people.</skos:example>

**Comment by stevenchalem** [comment]
Proposal: "Relates a thing to another thing whose goals or function it helps to bring about."

**Comment by uscholdm** [comment]
> Proposal: "Relates a thing to another thing whose goals or function it helps to bring about."

Fructose contributes to obesity, but does not have that as a goal or function, its a side effect.
Proposal: Relates a thing to another thing that the first thing plays a role in bringing about.
OR
The subject plays a role in bringing about the object.

**Comment by stevenchalem** [comment]
> > Proposal: "Relates a thing to another thing whose goals or function it helps to bring about."
> 
> Fructose contributes to obesity, but does not have that as a goal or function, its a side effect. Proposal: **Relates a thing to another thing that the first thing plays a role in bringing about.** OR **The subject plays a role in bringing about the object.**

Good point @uscholdm . I like the "subject plays a role" construction better, although we seem to be moving toward "Relates ..." for most object property definitions. In case we want to stick with the "Relates ..." pattern, I don't think there's any ambiguity in the simpler
**Relates a thing to another thing that it plays a role in bringing about.**

**Comment by uscholdm** [comment]
> I don't think there's any ambiguity in the simpler
> **Relates a thing to another thing that it plays a role in bringing about.**

There is no way to determine which thing "it" refers to, so it is ambiguous.  I generally like the relates style, but sometimes there is no nice clean wording, and the subject/object style is simpler to understand

**Comment by stevenchalem** [comment]
> > I don't think there's any ambiguity in the simpler
> > **Relates a thing to another thing that it plays a role in bringing about.**
> 
> There is no way to determine which thing "it" refers to, so it is ambiguous. I generally like the relates style, but sometimes there is no nice clean wording, and the subject/object style is simpler to understand

I'm fine with either
1. **Relates a thing to another thing that the first thing plays a role in bringing about.** or
2. **The subject plays a role in bringing about the object.**

**Comment by rjyounes** [comment]
DECISION: "Relates a thing to another thing whose goals or function it helps to bring about."

Add one or two examples, scope notes, etc.

**Comment by uscholdm** [comment]
> DECISION: "Relates a thing to another thing whose goals or function it helps to bring about."

The 'it' is amibgous, can say if it is the original or the other thing. Reword.  I think I made a concrete suggestion in the PR

**Comment by stevenchalem** [comment]
> > DECISION: "Relates a thing to another thing whose goals or function it helps to bring about."
> 
> The 'it' is amibgous, can say if it is the original or the other thing. Reword. I think I made a concrete suggestion in the PR

We discussed this at the dev meeting. While there is technically an ambiguity, the consensus was that the alternative is too clunky and nobody is actually likely to misread it.

Regardless, I have another proposal in case @rjyounes wants to weigh in on it:
“Relates one thing to another thing whose goals or functions the first thing helps to bring about.”

**Comment by rjyounes** [comment]
> “Relates one thing to another thing whose goals or functions the first thing helps to bring about.”

Not crazy about it - pretty awkward. I'm having trouble seeing the ambiguity of the proposed version. Can "it" really refer back to the second thing rather than the first, when "whose" refers to the second?

Another wording issue, though: you don't "bring about" a goal or function. You could realize or achieve a goal, fulfill a function, support a goal or function. Others?

**Comment by stevenchalem** [comment]
I updated the PR
